homeassistant/components/huawei_lte/__init__.py

```diff
@@ -82,9 +83,10 @@
     None,
     vol.Schema(
         {
+            vol.Optional(CONF_NAME): cv.string,
```

Comment:
Should we set a default name here too?

Reply:
No, because otherwise default will propagate to a "set in yaml" value and thus override whatever is set in config entry options.

Reply:
Only if the user uses config yaml and only if the name changes, right?

Reply:
Right.

Reply:
I noticed that the overriding doesn't actually work as intended. But it's nothing new related to this PR, it was already broken before. Filed #30221 for that.